tests/spread/integration/libarchive-tools/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for libarchive-tools
+
+execute: |
+  rootfs="$(install-slices libarchive-tools_bins)"
+
+  chroot "${rootfs}/" bsdcat --version
+  chroot "${rootfs}/" bsdcpio --version
+  chroot "${rootfs}/" bsdtar --version
+  chroot "${rootfs}/" bsdunzip --version
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  chroot "${rootfs}/" bsdtar -cf testfile.tar testfile.txt
+  if [ ! -f "${rootfs}/testfile.tar" ]; then 
+    echo "bsdtar failed to create archive: testfile.tar"
+    exit 1
```

Comment:
I would follow Canonical's suggestion in the `cpio` PR to also compare the content from the extracted version of the testfile with the content of the file that was used as the input.

Reply:
1. We should test extraction of the tar file in addition to creation of the tar file (similar to what we do for cpio).
2. We should check the content of the extracted version of the test file to confirm it's content matches our expectation (similar to what we do for cpio).